docs/encyclopedia/workers.mdx

```diff
@@ -190,6 +191,30 @@ Once an Activity Task finishes execution, the Worker responds to the Temporal Se
 - ActivityTaskTerminated
 - ActivityTaskTimedOut
 
+## What is a Nexus Operation Task? {#nexus-operation-task}
+
+A Nexus Operation Task provides the context needed to proceed with an Nexus Operation Task Execution.
+Nexus Operation Tasks largely represent the Nexus Operation Scheduled Event, containing the data needed to execute a Nexus Operation handler.
```

Comment:
A Nexus Task is a representation of a single Nexus request to start or cancel a Nexus Operation. The task includes information such as the Service and Operation names and other information required to process the Nexus request.
The Temporal Worker triggers the registered Operation handler based on the task's information.

Reply:
Thanks!